File: src/FooBarFrontend/Core/StringHelpers.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FooBarFrontend.Core
{
    public class StringHelpers
    {
        private static readonly Random _random = new Random();

        public static string RandomString()
        {
            return RandomString(_random.Next(8, 35));
        }

        public static string RandomString(int length)
        {
            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
            return new string(Enumerable.Repeat(chars, length)
              .Select(s => s[_random.Next(s.Length)]).ToArray());
        }
    }
}

File: src/FooBarAdminWeb/client/common/utils/string-helpers.ts
export class StringHelpers {
	static randomString(length: number): string {
		var text = "";
		var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
		for (var i = 0; i < length; i++) {
			text += possible.charAt(Math.floor(Math.random() * possible.length));
		}
		return text;
	}
}